lib/reporters/nyan.js

```diff
@@ -40,24 +41,24 @@ function NyanCat(runner) {
   this.trajectories = [[], [], [], []];
   this.trajectoryWidthMax = width - nyanCatWidth;
 
-  runner.on('start', function() {
+  runner.on(Runner.constants.RUNNER_EVENT_START, function() {
```

Comment:
Quite noisy in this state. Can we re-assign them all at top of file?

Reply:
do you mean e.g. `var RUNNER_EVENT_START = Runner.constants.RUNNER_EVENT_START`?

ugh, need es6 everywhere... somebody give me babel and webpack

Reply:
Yep thats what i mean...es6 everywhere 😍